examples/imagenet/train.py

```diff
@@ -88,24 +92,22 @@ def compute_metrics(logits, labels):
   return metrics
 
 
-def cosine_decay(lr, step, total_steps):
-  ratio = jnp.maximum(0., step / total_steps)
-  mult = 0.5 * (1. + jnp.cos(jnp.pi * ratio))
-  return mult * lr
-
-
-def create_learning_rate_fn(config: ml_collections.ConfigDict,
-                            base_learning_rate: float,
-                            steps_per_epoch: int):
-
-  def step_fn(step):
-    epoch = step / steps_per_epoch
-    lr = cosine_decay(base_learning_rate,
-                      epoch - config.warmup_epochs,
-                      config.num_epochs - config.warmup_epochs)
-    warmup = jnp.minimum(1., epoch / config.warmup_epochs)
-    return lr * warmup
-  return step_fn
+def create_learning_rate_fn(
```

Comment:
This is very nice! I think the optax learning rate fns are much easier to reason about that the ones we've been using so far.

Reply:
Thanks I am glad you like them :)